packages/world/ts/node/render-solidity/renderWorldInterface.ts

```diff
@@ -20,7 +20,7 @@ export function renderWorldInterface({
   const imports = [
     ...(interfaceName === "IBaseWorld"
       ? [
-          { symbol: "IStore", path: `${storeImportPath}/IStore.sol` },
+          { symbol: "IStoreKernel", path: `${storeImportPath}/IStoreKernel.sol` },
```

Comment:
Wonder if this breaks functions that expect `IStore` as input, which `IBaseWorld` previously matched but now doesn't anymore? But not sure how common that is, if it happens would be easy enough to work around by casting to `IStore`. If CI is happy, I'm happy.

Reply:
just discovered that an fixing it now